src/Docker.PowerShell/Docker.psm1

```diff
@@ -6,6 +6,7 @@
 # the module is in.
 #
 
+$paramBackup = $PSDefaultParameterValues.Clone()
```

Comment:
It's in the published guidance from the powershell team:
https://msdn.microsoft.com/en-us/powershell/gallery/psget/module/modulewithpseditionsupport#option-1-packaging-a-module-for-targeting-multiple-versions-and-multiple-editions-of-powershell